scripts/lib/upgrade-zulip-stage-2

```diff
@@ -150,7 +150,15 @@ if glob.glob("/usr/share/postgresql/*/extension/tsearch_extras.control"):
             "su",
             "postgres",
             "-c",
-            'psql -v ON_ERROR_STOP=1 zulip -c "DROP EXTENSION IF EXISTS tsearch_extras;"',
+            "psql",
+            "-v",
+            "ON_ERROR_STOP=1",
+            "-d",
+            get_config(config_file, "postgresql", "database_name"),
+            "-u",
+            get_config(config_file, "postgresql", "database_user"),
+            "-c",
+            "DROP EXTENSION IF EXISTS tsearch_extras;",
```

Comment:
This is probably unnecessary, in that it's code for a legacy state.  But reading this gives me an idea for a useful prep commit -- we should just make a `run_sql_as_postgres` function in `zulip_tools.py`, as a prep commit, that passes all the parameters here and just requires the SQL script, and then this locus and potentially other calls can use that.